package.json

```diff
@@ -17,9 +17,7 @@
     "rfc4122"
   ],
   "license": "MIT",
-  "bin": {
-    "uuid": "./dist/bin/uuid"
-  },
+  "bin": "./dist/bin/uuid",
```

Comment:
What’s the effect of this change?

Reply:
Nothing, since [both forms are allowed](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#bin).

I'm not actually sure why this changed.  I noticed it eariler and assumed it was eslint or prettier being clever, but that doesn't appear to be the case. 🤷   I'll revert since the docs kind of make it sound like the object-form is preferred.